### DISCUSSION TITLE
Anyone know why tridactyl suddenly stops working on some sites after a while?

### DISCUSSION THREAD

**Discussion by garyfish78** [tridactyl/tridactyl#5082] [Q&A]
I've been using this software for a few days now, and it's excellent, but one recurring problem is that the tool sometimes de-loads on a webpage randomly, and my keybinds are now once again performing their built in function. It can be incredibly annoying. I am wondering if I have my browser configured incorrectly or something, because I have not seen mention of this anywhere else.

Anyways, help/advice would be appreciated!

**Comment by bovine3dom** [comment]
Tridactyl doesn't get loaded into iframes which are like mini sub-webpages within a page. Stuff like YouTube videos use them.

You should be able to get focus back to Tridactyl by pressing ctrl+comma

**Reply by garyfish78** [reply]
I see. Thank you for responding so quickly, I didn't notice you before I went to bed. Is there any way to load tridactyl into iframes automatically?

**Reply by bovine3dom** [reply]
As a user, no.

There's an issue open for rewriting Tridactyl to load into iframes here https://github.com/tridactyl/tridactyl/issues/412
